.eslintrc

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "@logux/eslint-config/esm"
+  "extends": "@logux/eslint-config/esm",
+  "overrides": [
+    {
+      "files": "**/**.js",
+      "rules": {
+        "import/extensions": "on",
```

Comment:
Let's remove ESLint changes.

1. We do not need to enable first rule since it was already enabled in main config
2. I use `let` instead of `const` and it will be easier for me to support